### PR TITLE
FuckedHard18 Scraper

### PR DESCRIPTION
**Pull request by Caleripo**
Adds a FuckedHard18.com scraper as requested in #73.
The scraper is tested and validated. However, this is the first scaper that I add to the repo so if someone could double-check that would be appreciated.

### CODE REVIEW

File: SCRAPERS-LIST.md [hunk 1]
```diff
@@ -416,6 +416,7 @@ freeusemilf.com|PaperStreetMedia.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 freshoutofhighschool.com|NewSensationsNetworkSites.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 frolicme.com|FrolicMe.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 fuckedfeet.com|VNAGirls.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+fuckedhard18.com|FuckedHard18.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```
Comment (Belleyy) [inline SCRAPERS-LIST.md:419]: Your scraper only support scene so you should remove the ❌ for Gallery & Movies
`fuckedhard18.com|FuckedHard18.yml|:heavy_check_mark:|:x:|-|-|-|-`

File: scrapers/FuckedHard18.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - action: scrapeXPath
+    url:
+      - fuckedhard18.com/membersarea/gallery.php
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &name
+        selector: //a[@class="model_category_link"][contains(@href,'sets.php')]/text()
+      Details:
```
Comment (Belleyy) [inline scrapers/FuckedHard18.yml:12]: Remove it if it's not used.

File: scrapers/FuckedHard18.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      Date:
+        selector: //td[@class="date"]/text()
+        postProcess:
+          - replace:
+            - regex: Added 
+              with: ""
+          - parseDate: 01/02/2006 
+      Image: 
+        selector: //a[@class="jqModal"]/img[@class="thumbs"]/@src[1]
```
Comment (bnkai) [inline scrapers/FuckedHard18.yml:21]: This returns all the `idividual clip` images. ( of which stash needs only 1 so gets the  first)
If we only want the first (i also cant find an alternative for a better quality image), to be more exact
```suggestion
        selector: (//a[@class="jqModal"]/img[@class="thumbs"])[1]/@src
```
should be used instead

**Comment by Caleripo** [comment]
Thanks a lot guys! Still a lot to learn for me but trying to contribute to this project!

File: scrapers/FuckedHard18.yml (hunk 3 above)
Comment (Caleripo) [reply scrapers/FuckedHard18.yml:21]: Thanks! Yes, unfortunately the data on the website is quite poor also for title and tags. In most cases the thumb generated by stash is better.

**Pull request merged**